configuration/csv-to-docs.md

```diff
@@ -1,32 +1,276 @@
+    "_id": "e8f9739a-5d37-5b1e-be3c-a571b2c2409b"
+  },
+  "is_name_generated": "FALSE",
+  "name": "HC1",
+  "reported_date": 1544031155715,
+  "_id": "8606a91a-f454-56e3-a089-0b686af3c6b7"
+}
+```
+
+**csv/person.csv**
+
+```
+reference_id:excluded,parent:place WHERE reference_id=COL_VAL,name,phone,sex,role,reported_date,patient_id
+p_hc1,health_center_1,Bob Johnson 1,+16143291527,male,manager,1552494835669,60951
+p_hc2,health_center_1,Bob Johnson 2,+16143291528,male,manager,1552494835669,60951
+
+```
+Generated json doc for the person
+```
+{
+  "type": "person",
+  "parent": {
+    "type": "health_center",
+    "parent": {
+      "type": "district_hospital",
+      "parent": "",
+      "is_name_generated": "false",
+      "name": "District1",
+      "external_id": "",
+      "notes": "",
+      "geolocation": "",
+      "reported_date": 1544031155715,
+      "_id": "e8f9739a-5d37-5b1e-be3c-a571b2c2409b"
+    },
+    "is_name_generated": "FALSE",
+    "name": "HC1",
+    "reported_date": 1544031155715,
+    "_id": "8606a91a-f454-56e3-a089-0b686af3c6b7"
+  },
+  "name": "Bob Johnson 1",
+  "phone": "+16143291527",
+  "sex": "male",
+  "role": "manager",
+  "reported_date": "1552494835669",
+  "patient_id": "60951",
+  "_id": "65c52076-84c5-53a2-baca-88e6ec6e0875"
+}
+
+```
+
+**csv/users.csv**
+```
+username,password,roles,phone,contact:person WHERE reference_id=COL_VAL,place:GET _id OF place WHERE reference_id=COL_VAL
+ac1,Secret_1,district_admin:red1,+123456789,p_hc1,health_center_1
+ac2,Secret_1,district_admin:supervisor,+123456789,p_hc2,health_center_1
+ac3,Secret_1,district_admin,+123456789,p_hc3,health_center_1
+ac4,Secret_1,district_admin,+123456789,p_hc4,health_center_1
+
+```
+This will generate the `users.csv` file in the working directory which is used by the `create-users` action. The contact and place fields should be resolved to the actual UUIDs.
+
+```
+p_hc1"username","password","roles","contact","phone","place"
+"ac1","Secret_1","district_admin:red1","65c52076-84c5-53a2-baca-88e6ec6e0875","+123456789","8606a91a-f454-56e3-a089-0b686af3c6b7"
+"ac2","Secret_1","district_admin:supervisor","b7d0dbd5-beeb-52a8-8e4c-513d0baece8e","+123456789","8606a91a-f454-56e3-a089-0b686af3c6b7"
+```
+
+
+### Using CSV Files on Google Drive
+
+In your project home directory create a new json file named `csvs-on-google-drive.json`. 
+
+The keys should be the name of the CSV file stored locally, and for the value use the google id for the document. The name of the file must match what the file is generating a doc for. See the documentation [here](https://github.com/medic/medic-conf#csv-file-name) for naming files. 
+
+```
+{
+    "person.clinic.csv":"google_drive_ID",
+}
+```
+
+To fetch the files from Google Drive run the command `medic-conf fetch-csvs-from-google-drive`. This will download the CSV files in the json document created above and place them into a folder named `csv`.
+
+#### Google Drive Authentication
```

Comment:
This should be before we use it above, as it's the first step.

Reply:
I think this is addressed partly in https://github.com/medic/medic-docs/commit/71ff030fe5f6eb8296ba5992a742a1896df3c1ef. I don't think the steps need to be chronological since the intention is for this document to be less of a tutorial and more of a reference documentation. As such the general concept is described, and then more detailed info is provided since it may not be relevant to all.